octopoes/octopoes/repositories/ooi_repository.py

```diff
@@ -292,9 +293,17 @@ def list_oois(
         limit: int = 20,
         scan_levels: set[ScanLevel] = DEFAULT_SCAN_LEVEL_FILTER,
         scan_profile_types: set[ScanProfileType] = DEFAULT_SCAN_PROFILE_TYPE_FILTER,
+        search_string: str | None = None,
     ) -> Paginated[OOI]:
         types = to_concrete(types)
 
+        search_statement = (
+            f"""[?e :xt/id ?id]
+                                [(clojure.string/includes? ?id \"{search_string}\")]"""
```

Comment:
This needs escaping against clojure injection.

Reply:
Done